### ISSUE TITLE
[Bug Report][2.5.6] VTabs scroll behaviour breaks when using keyboard and then mouse

### ISSUE THREAD

**Issue by yungxramoz** [vuetifyjs/vuetify#13925]
### Environment
**Vuetify Version:** 2.5.6
**Vue Version:** 2.6.14
**Browsers:** Chrome 91.0.4472.124
**OS:** Windows 10

### Steps to reproduce
- Set focus on the first Tab by clicking it
- Navigate thru the Tabs to to the last one (Item 15) by using Tab-key
- Bug: Hit Enter Key => Scrolls to far
- Bug: Now using the Mouse to scroll backwards then selecting a tab by click scrolls somewhere and it's no possible to scroll to the first tab

### Expected Behavior
All tabs are still accessible and the scroll position stays or centers the selected tab.

### Actual Behavior
Selecting a tab scrolls to wrong position and some tabs are not accessible anymore.

### Reproduction Link
<a href="https://codepen.io/Yungxramoz/pen/zYwNXqZ" target="_blank">https://codepen.io/Yungxramoz/pen/zYwNXqZ</a>
